pyaedt/modeler/PrimitivesMaxwellCircuit.py

```diff
@@ -0,0 +1,251 @@
+from pyaedt.generic.general_methods import aedt_exception_handler
+from pyaedt.modeler.PrimitivesCircuit import CircuitComponents
+
+
+class MaxwellCircuitComponents(CircuitComponents):
+    """MaxwellCircuitComponents class.
+
+    This class is for managing all circuit components for MaxwellCircuit.
+
+    Parameters
+    ----------
+    parent :
```

Comment:
I don't think that this parameter is used in the initializer.

Reply:
removed